File: src/API/js/module.js
const fs = require('fs');

function get(deviceName) {
    const data = fs.readFileSync('info.txt', 'utf8');
    const lines = data.split('\n');
    for (let line of lines) {
        if (line.startsWith(`${deviceName}:"`)) {
            return line.split(':')[1].slice(1, -1);
        }
    }
    return "Device not found";
}

File: src/API/java/module.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.util.stream.Stream;

public class Lilia {
    public static String get(String deviceName) {
        try (Stream<String> lines = Files.lines(Paths.get("info.txt"))) {
            return lines
                    .filter(line -> line.startsWith(deviceName + ":"))
                    .findFirst()
                    .map(line -> line.split(":")[1].replaceAll("^\"|\"$", ""))
                    .orElse("Device not found");
        } catch (IOException e) {
            e.printStackTrace();
            return "Error reading file";
        }
    }

    public static void main(String[] args) {
        System.out.println(get("cpu"));
    }
}
